src/main/java/core/basesyntax/ColorSupplire.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplire {

```

Comment:
Class name should be corrected to `ColorSupplier` instead of `ColorSupplire` to follow Java naming conventions and avoid confusion.